ThunderCloud/CollectionCell.swift

```diff
@@ -236,3 +240,14 @@ extension CollectionCell : UICollectionViewDataSource {
         return collectionCell
 	}
 }
+
+extension CollectionCell: ScrollOffsetManagable {
```

Comment:
When conforming a `class` to a `protocol` perhaps it's best to declare the conformance on the `class` declaration (in the main file) rather than an `extension`.
Since class support inheritance, a subclass might want to override the protocol conformance implementation.
If done in an extension we won't be able to override (maybe with `@objc` - but meh)

In the case of `struct`, `enum` as there's no inheritance this approach is defo cleaner to read and write! :)

Reply:
That's a very good shout! Will do so now :)